akka-docs/src/main/paradox/stream/operators/Sink/asPublisher.md

```diff
@@ -12,4 +12,32 @@ Integration with Reactive Streams, materializes into a `org.reactivestreams.Publ
 
 ## Description
 
-TODO: We would welcome help on contributing descriptions and examples, see: https://github.com/akka/akka/issues/25646
+This method gives you the capability to publish the data from the `Sink` through a Reactive Streams [Publisher](http://www.reactive-streams.org/reactive-streams-1.0.3-javadoc/org/reactivestreams/Publisher.html).
+Generally, in Akka Streams a `Sink` is considered a subscriber, which consumes the data from source. To integrate with other Reactive Stream implementations `Sink.asPublisher` provides a `Publisher` materialized value when run.
+Now, the data from this publisher can be consumed by subscribing to it. We can control if we allow more than one downstream subscriber from the single running Akka stream through the `fanout` parameter.
+In Java 9, the Reactive Stream API was included in the JDK, and `Publisher` is available through [Flow.Publisher](https://docs.oracle.com/javase/9/docs/api/java/util/concurrent/Flow.Publisher.html).
+Since those APIs are identical but exist at different package namespaces and does not depend on the Reactive Streams package a separate publisher sink for those is available 
+through @scala[`akka.stream.scaladsl.JavaFlowSupport.Sink#asPublisher`]@java[`akka.stream.javadsl.JavaFlowSupport.Sink#asPublisher`].
+
+
+## Example
+
+In the example I'm using a source and then creating a Publisher. After that, we see that when `fanout` is true multiple subscribers can subscribe to it, 
+but when it is false only the first subscriber will be able to subscribe and others will be rejected.
```

Comment:
I don't know if we are consistent in using first person singular or plural. 

I do have a preference for using 'we' for this kind of text. And in that case, the same paragraph is using both forms. 

I would suggest

```suggestion
In the example we are using a source and then creating a Publisher. After that, we see that when `fanout` is true multiple subscribers can subscribe to it, 
but when it is false only the first subscriber will be able to subscribe and others will be rejected.
```

WDYT?

Reply:
It was by mistake I think, will fix it, and this can be merged